lib/hyrax/arkivo/actor.rb

```diff
@@ -50,20 +49,22 @@ def destroy_work(work)
 
       private
 
-        def reset_metadata(work)
-          work.resource_type = []
-          work.title = []
-          work.rights = []
-          work.keyword = []
-          work.creator = []
-          work.description = []
-          work.publisher = []
-          work.date_created = []
-          work.based_near = []
-          work.identifier = []
-          work.related_url = []
-          work.language = []
-          work.contributor = []
+        def default_attributes
```

Comment:
@jcoyne Please add the comment in default_attributes about the keys should be strings so they merge with arkivio